config/eslintrc_react.js

```diff
@@ -112,6 +114,7 @@ module.exports = {
 
         // JSX-specific rules
         'react/jsx-boolean-value': [2, 'always'], // Enforce to specify html's boolean type attribute.
+        'react/jsx-child-element-spacing': 0,
```

Comment:
👍 

This rule set also enables [`jsx-no-literals`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-literals.md), so I feel there is no reason to enable this.